devito/base.py

```diff
@@ -6,8 +6,10 @@
 
 from devito.backends import _BackendSelector
 from devito.grid import Grid
-import devito.types as types
-import devito.function as function
+import devito.functions.basic as types
+import devito.functions.dense as function
+import devito.functions.sparse as sparse
```

Comment:
can you not just ``import devito.function as function`` and simply just that, instead of ``function, sparse, constant``?